src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -195,62 +163,29 @@ export const ArtworkFilterRefetchContainer = createRefetchContainer(
       </Subscribe>
     )
   },
-  {
-    artist: graphql`
-      fragment ArtworkFilter_artist on Artist
-        @argumentDefinitions(
-          medium: { type: "String", defaultValue: "*" }
-          major_periods: { type: "[String]" }
-          partner_id: { type: "ID" }
-          for_sale: { type: "Boolean" }
-          aggregations: {
-            type: "[ArtworkAggregation]"
-            defaultValue: [MEDIUM, TOTAL, GALLERY, INSTITUTION, MAJOR_PERIOD]
-          }
-          sort: { type: "String", defaultValue: "-partner_updated_at" }
-        ) {
-        id
-        filtered_artworks(
-          aggregations: $aggregations
-          medium: $medium
-          major_periods: $major_periods
-          partner_id: $partner_id
-          for_sale: $for_sale
-          size: 0
-          sort: $sort
-        ) {
-          aggregations {
-            slice
-            counts {
-              name
-              count
-              id
-            }
+  graphql`
+    fragment ArtworkFilter_artist on Artist
+      @argumentDefinitions(
+        medium: { type: "String", defaultValue: "*" }
+        major_periods: { type: "[String]" }
+        partner_id: { type: "ID" }
+        for_sale: { type: "Boolean" }
+        aggregations: {
+          type: "[ArtworkAggregation]"
+          defaultValue: [MEDIUM, TOTAL, GALLERY, INSTITUTION, MAJOR_PERIOD]
+        }
+      ) {
+      id
+      filtered_artworks(aggregations: $aggregations, size: 0) {
+        aggregations {
+          slice
+          counts {
+            name
+            id
           }
-          ...ArtworkFilterArtworkGrid_filtered_artworks
         }
       }
-    `,
-  },
-  graphql`
-    query ArtworkFilterRefetchQuery(
-      $artistID: String!
-      $medium: String
-      $major_periods: [String]
-      $partner_id: ID
-      $for_sale: Boolean
-      $sort: String
-    ) {
-      artist(id: $artistID) {
-        ...ArtworkFilter_artist
-          @arguments(
-            medium: $medium
-            major_periods: $major_periods
-            partner_id: $partner_id
-            for_sale: $for_sale
-            sort: $sort
-          )
-      }
+      ...ArtworkFilterRefetch_artist
```

Comment:
@mzikherman One thing to keep in mind is that _maybe_ this ends up calling the filter endpoint multiple times and so maybe MP needs a little optimisation?

Reply:
Yea, this could be a follow up! I'd love to get this merged in the meantime since it's an improvement (will rename away from `gridStuff`).

Also we still have a state issue where when you select a filter, you get 2 simultaneous network requests to Metaphysics...

Reply:
Let me know if you need help figuring that double fetch out.